pyproject.toml

```
@@ -1,16 +1,46 @@
 [build-system]
-requires = ["setuptools>=61.2",
-            "setuptools-scm",
-            "wheel >= 0.29.0"]  # ought to mirror 'requirements/build.txt'
 build-backend = "setuptools.build_meta"
+requires = [
+  "setuptools>=61.2",
+  "setuptools-scm",
+  "wheel>=0.29",
+]  # ought to mirror 'requirements/build.txt'
```

```suggestion
]
```

Oh look, someone forgot to remove this part! 😅

---

And looks like someone forgot to point this out in code review too! 😅